Extension/c_cpp_properties.schema.json

```diff
@@ -142,6 +142,11 @@
         "version": {
             "type": "integer",
             "description": "Version of the configuration file. This property is managed by the extension. Please do not change it."
+        },
+        "enableConfigurationSquiggles": {
+            "type": "boolean",
+            "default": true,
+            "description": "Set to false to disable squiggles for c_cpp_properties.json."
```

Comment:
"Controls whether the extension will report errors detected in c_cpp_properties.json"